DfE.FindInformationAcademiesTrusts/Pages/Schools/Overview/Details.cshtml.cs

```diff
@@ -21,4 +23,30 @@ public static string SubPageName(SchoolCategory schoolCategory)
             _ => throw new ArgumentOutOfRangeException(nameof(schoolCategory))
         };
     }
+
+    public SchoolOverviewServiceModel SchoolOverviewModel { get; private set; } = null!;
+
+    public string? GetInformationAboutSchoolsLink { get; set; }
```

Comment:
These three links will never be null if the page renders so we could make them non-nullable and remove the null handling from the razor code